esmvalcore/preprocessor/_volume.py

```diff
@@ -104,7 +104,7 @@ def extract_volume(
     return cube.extract(z_constraint)
 
 
-def calculate_volume(cube: Cube) -> da.core.Array:
+def calculate_volume(cube: Cube) -> np.ndarray | da.Array:
```

Comment:
```suggestion
def calculate_volume(cube: Cube) -> np.ndarray | dask.array.Array:
```

Reply:
egh those should be full names of modules not `import numpy as cow; obj: cow.ndarray` :grin: